### DISCUSSION TITLE
Question: How does "Merge records" work?

### DISCUSSION THREAD

**Discussion by burtek** [Serg-Norseman/GEDKeeper#554] [Q&A]
I'm not sure how the "Merge records" works. Can I get any clarity on what those buttons do and what the outcome of merge, based on this example?

![image](https://github.com/Serg-Norseman/GEDKeeper/assets/2940958/aa1760e8-dcc6-4d70-b45a-be09fbfc9b14)

Specifically:
 - What do `<<<` and `>>>` do?
 - What do `Search` and `Skip` do in this context?

**Comment by Serg-Norseman** [comment]
"Search" - launches a search with the specified settings until the first pair of duplicates found. “Skip” - go to the next pair, “<<<” and “>>>” into which record to merge the other.
